src/apps/users/tests/test_username_len.py

```diff
@@ -22,5 +23,16 @@ def test_username_len(self):
         with transaction.atomic():
             with pytest.raises(DataError):
                 self.users.append(User.objects.create_user(username="1234567890123456789012345678901234567890123456789012345678901", password="test"))
+        self.users.append(User.objects.create_user(username="123", password="test"))
+        self.users.append(User.objects.create_user(username="quertyexample.com", password="test"))
         for user in self.users:
             user.full_clean()
+        baduser0 = User.objects.create_user(username="12", password="test")
+        baduser1 = User.objects.create_user(username="querty@example.com", password="test")
+        self.users.append(baduser0)
+        self.users.append(baduser1)
```

Comment:
maybe add a test for username "a" (min length)

Reply:
Done, thanks.